test-harness/modules/generic_test_code.py

```diff
@@ -8,7 +7,52 @@
 log = logging.getLogger(__name__)
 
 
-def generic_unknown_user_is_forbidden_test(ar, auth_header, path):
+def ping_mesos_agent(ar,
```

Comment:
Got it! I just don't see this function being actually used anywhere in the code.